utils/try_require.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// Performs a safe require to the given path
+// and returns null in case of failure
```

Comment:
What does it return in the case of success? Seems like we're doing a truthy test in the caller -- should we make that explicit?

Reply:
In case of success, it loads the library and returns the available objects that were exposed by `exports`. But I agree that the way it is being used is to check if the `require` loads properly without errors and in the case of success the return object is not used. 
What do you suggest to make it explicit?

Reply:
Signature seems fine -- maybe just mention the return value in the success case here.

Reply:
I've removed `tryRequire` see next comment.